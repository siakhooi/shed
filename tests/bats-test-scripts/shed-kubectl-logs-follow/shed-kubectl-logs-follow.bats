setup(){
    load '../common-setup'
    common_setup
}
@test "shed-kubectl-logs-follow" {
    run shed-kubectl-logs-follow
    assert_success
    cat_expected "shed-kubectl-logs-follow-usage" | assert_output -
}
@test "shed-kubectl-logs-follow xxx yyy zzz" {
    run shed-kubectl-logs-follow xxx yyy zzz
    assert_success
    cat_expected "shed-kubectl-logs-follow-usage" | assert_output -
}
@test "shed-kubectl-logs-follow xxx" {
    run shed-kubectl-logs-follow xxx
    assert_failure 1
    cat_expected "shed-kubectl-logs-follow-xxx" | assert_output -
}
@test "shed-kubectl-apply pod-busybox-loop; shed-kubectl-logs-follow pod-busybox-loop" {
    skip 'Unable to use a TTY - input is not a terminal or the right kind of file'
    shed-kubectl-apply pod-busybox-loop
    run bash -ic "echo 'exit' |shed-kubectl-logs-follow busybox-looping "
    assert_success
    cat_expected "shed-kubectl-logs-follow" | assert_output -
    shed-kubectl-delete pod-busybox-loop
}
@test "shed-kubectl-apply pod-busybox-loop; shed-kubectl-logs-follow pod-busybox-loop busybox-1" {
    skip 'Unable to use a TTY - input is not a terminal or the right kind of file'
    shed-kubectl-apply pod-busybox-loop
    run bash -ic "echo 'exit' |shed-kubectl-logs-follow busybox-looping busybox-1"
    assert_success
    cat_expected "shed-kubectl-logs-follow-container" | assert_output -
    shed-kubectl-delete pod-busybox-loop
}
