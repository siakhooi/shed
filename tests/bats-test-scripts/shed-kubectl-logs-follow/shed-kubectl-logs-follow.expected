+ kubectl --context kube-dev-1 --namespace ns-application logs -f --tail=0 busybox-looping
